src/rgw/rgw_user.cc

```diff
@@ -725,7 +725,8 @@ static void dump_user_info(Formatter *f, RGWUserInfo &info,
   f->dump_string("email", info.user_email);
   f->dump_int("suspended", (int)info.suspended);
   f->dump_int("max_buckets", (int)info.max_buckets);
-
+  if (info.system)
+    f->dump_string("system", "true");
```

Comment:
I think you could use `Formatter::dump_bool()`

Reply:
@joscollin thanks . addressed!